colin/cli/colin.py

```diff
@@ -0,0 +1,29 @@
+import click as click
+
+from ..core.colin import run
+
+CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
+
+
+@click.command(context_settings=CONTEXT_SETTINGS)
+@click.argument('target', type=click.STRING)
+@click.option('--config', '-c', type=click.STRING,
+              help="Configuration name.")
+def cli(target, config):
+    results = run(name_of_target=target,
+                  config_name=config)
+
+    for r in results.results:
+        click.secho(str(r), err=r.ok, fg='green' if r.ok else 'red')
+        if not r.ok:
+            click.secho("   -> {}\n"
```

Comment:
good enough for PoC, [we need to make this better in future](https://github.com/user-cont/colin/issues/14)